python/lsst/qserv/tests/benchmark.py

```diff
@@ -145,6 +145,9 @@ def runQueries(self, stopAt=MAX_QUERY):
                                       str(pragmas))
                     column_names = 'noheader' not in pragmas
                     sqlInterface.execute(qText, outFile, column_names)
+                    if 'sortresult' in pragmas:
+                        sortcmd = 'sort %s -o %s' % (outFile, outFile)
+                        os.system(sortcmd)
```

Comment:
This works for me:
```python
if 'sortresult' in pragmas:
    with open(outFile,"r+") as f:
        sortedLines = sorted(f.readlines())
        f.seek(0)
        f.writelines(sortedLines)
```
No need to call `f.truncate()` because we are writing exactly what we read.

My comment on the check was relating to the use of `os.system()`. Checking the return value might have taken 2 lines though with the `if` and the `raise`.